src/Service/VendorService/VendorImageValidatorService.php

```diff
@@ -45,7 +45,12 @@ public function validateRemoteImage(VendorImageItem $item): void
             $lastModifiedArray = $head->getHeader('Last-Modified');
 
             $timezone = new \DateTimeZone('UTC');
-            $lastModified = \DateTime::createFromFormat('D, d M Y H:i:s \G\M\T', array_shift($lastModifiedArray), $timezone);
+            if (empty($lastModifiedArray)) {
+                // Not all server send last modified headers so fallback to now.
+                $lastModified = new \DateTime('now', $timezone);
+            } else {
+                $lastModified = \DateTime::createFromFormat('D, d M Y H:i:s \G\M\T', time(), $timezone);
```

Comment:
Shouldn't this line use `array_shift($lastModifiedArray)` and not `time()`?

Reply:
Fixed :-)